contracts/src/v0.8/llo-feeds/v0.4.0/DestinationVerifierProxy.sol

```diff
@@ -37,7 +37,7 @@ contract DestinationVerifierProxy is IDestinationVerifierProxy, ConfirmedOwner,
     bytes calldata payload,
     bytes calldata parameterPayload
   ) external payable returns (bytes memory) {
-    return s_verifier.verify(payload, parameterPayload, msg.sender);
+    return s_verifier.verify{value: msg.value}(payload, parameterPayload, msg.sender);
```

Comment:
@Fletch153 tiny fix to address the native errors we saw

Reply:
There should be 4 places we need to update:

s_verifier.verify
s_verifier.verifyBulk
s_feeManager.verify
s_feeManager.verifyBulk